src/System Application/App/Performance Profiler/src/PerformanceProfiles.Page.al

```diff
@@ -0,0 +1,208 @@
+                    ToolTip = 'Specifies the ID of the client session that was profiled.';
+                }
+                field("Schedule ID"; Rec."Schedule ID")
+                {
+                    Caption = 'Schedule ID';
+                    ToolTip = 'Specifies the ID of the schedule that was used to profile the activity.';
+                    TableRelation = "Performance Profile Scheduler"."Schedule ID";
+                    DrillDown = true;
+
+                    trigger OnDrillDown()
+                    var
+                        PerfProfileSchedule: Record "Performance Profile Scheduler";
+                        PerfProfileScheduleCard: Page "Perf. Profiler Schedule Card";
+                    begin
+                        if not PerfProfileSchedule.Get(Rec."Schedule ID") then
+                            exit;
+
+                        PerfProfileScheduleCard.SetRecord(PerfProfileSchedule);
+                        PerfProfileScheduleCard.Run();
+                    end;
+                }
+            }
+        }
+    }
+
+    actions
+    {
+        area(Promoted)
+        {
+            actionref(OpenProfiles; "Open Profile")
+            {
+            }
+
+            actionref(Refresh; RefreshPage)
+            {
+            }
+
+            actionref(DownloadProfile; Download)
+            {
+
+            }
+        }
+
+        area(Navigation)
+        {
+            action("Open Profile")
+            {
+                ApplicationArea = All;
+                Image = Setup;
+                Caption = 'Open Profile';
+                ToolTip = 'Open profiles for the scheduled session';
+                Enabled = Rec."Activity ID" <> '';
+                ShortcutKey = 'Return';
+
+                trigger OnAction()
+                var
+                    ProfilerPage: Page "Performance Profiler";
+                    ProfileInStream: InStream;
+                begin
+                    Rec.CalcFields(Profile);
+                    Rec.Profile.CreateInStream(ProfileInStream);
+                    ProfilerPage.SetData(ProfileInStream);
+                    ProfilerPage.Run();
+                end;
+            }
+
+            action(RefreshPage)
+            {
+                ApplicationArea = All;
+                Image = Refresh;
+                Caption = 'Refresh';
+                ToolTip = 'Refresh the profiles for the schedule.';
+
+                trigger OnAction()
+                begin
+                    Update();
+                end;
+            }
+
+            action(Download)
+            {
+                ApplicationArea = All;
+                Image = Download;
+                Enabled = Rec."Activity ID" <> '';
+                Caption = 'Download';
+                ToolTip = 'Download the performance profile file.';
+
+                trigger OnAction()
+                var
+                    SampPerfProfilerImplCodeunit: Codeunit "Sampling Perf. Profiler Impl.";
```

Comment:
```suggestion
                    SamplingPerfProfilerImpl: Codeunit "Sampling Perf. Profiler Impl.";
```

Reply:
no need to add the type to the variable name

Reply:
Fixed